include/seqan3/alignment/pairwise/policy/affine_gap_policy.hpp

```diff
@@ -58,73 +52,96 @@ class affine_gap_policy
     //!\}
 
     /*!\brief Computes the score of the current cell.
-     * \tparam  matrix_entry_type The type of the current cell.
-     * \tparam  cache_type        The type of the cache.
+     * \tparam  cell_t     The type of the current cell.
```

Comment:
Can you document what the `cell_t` looks like? I know you avoid concepts here but for clarity it would be good to document that this is a tuple (?) of references to the corresponding matrices. 

Copy and paste this for all `typename cell_t`